cmd/pdcli/login/cmd.go

```diff
@@ -13,31 +13,168 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
+
 package login
 
 import (
+	"bufio"
+	"errors"
 	"fmt"
+	"os"
 
+	"github.com/PagerDuty/go-pagerduty"
+	"github.com/openshift/pagerduty-short-circuiter/pkg/config"
+	"github.com/openshift/pagerduty-short-circuiter/pkg/pdcli"
 	"github.com/spf13/cobra"
 )
 
-// pdcli/login/cmdCmd represents the pdcli/login/cmd command
+const APIKeyURL = "https://support.pagerduty.com/docs/generating-api-keys#section-generating-a-general-access-rest-api-key"
+
+var loginArgs struct {
+	apiKey string
+}
+
 var Cmd = &cobra.Command{
 	Use:   "login",
-	Short: "A brief description of your command",
-	Long: `A longer description that spans multiple lines and likely contains examples
-and usage of using your command. For example:
-`,
-	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println("logged in")
-	},
+	Short: "Login to the PagerDuty CLI",
+	Long: `Running the pdcli login command will send a request to PagerDuty REST API provided a valid API key.
+The PagerDuty REST API supports authenticating via the user API token.`,
```

Comment:
Long:  We don't have to explain internal implementations to the end user, think about what more information, you want to tell the user, if there is none, you can leave this part.